datameta/security/__init__.py

```diff
@@ -103,16 +103,43 @@ def check_password_by_hash(pw, hashed_pw):
 
 def hash_token(token):
     """Hash a token and return the unsalted hash"""
-    hashed_token =  hashlib.sha256(token.encode('utf-8')).hexdigest()
+    hashed_token = hashlib.sha256(token.encode('utf-8')).hexdigest()
     return hashed_token
 
 
+def register_failed_login_attempt(db, user):
+    """ Registers a failed login attempt and disables user if this has happened too often in the last hour."""
+    BLOCK_AFTER_N_FAILED_LOGINS = 5
+
+    now = datetime.now()
```

Comment:
We're storing all datetimes in the database in UTC
```suggestion
    now = datetime.utcnow()
```